gce_workload_cert_refresh/main.go

```diff
@@ -262,6 +275,20 @@ func refreshCreds() error {
 		return fmt.Errorf("Error writing config_status: %v", err)
 	}
 
+	// Handles the edge case where the config values provided at VM creation may be invalid. This ensures
+	// that the symlink directory exists and contains the config_status to surface config errors to the VM.	
+	if _, err := os.Stat(symlink); os.IsNotExist(err) {
+		logger.Infof("Creating new symlink %s", symlink)
+
+		if err := os.Symlink(contentDir, symlink); err != nil {
+			return fmt.Errorf("Error creating symlink link: %v", err)
+		}
+		// Write config_status to the newly created symlink immediately.
+		if err := os.WriteFile(fmt.Sprintf("%s/config_status", symlink), certConfigStatus, 0644); err != nil {
```

Comment:
it's not clear why we have now a *third* write of config_status. we should also never write through the symlink. in this codebase, we always reference the per-run directory, and we do symlink operations separately.